airbyte-scheduler/persistence/src/test/java/io/airbyte/scheduler/persistence/DefaultJobPersistenceTest.java

```diff
@@ -1002,6 +1007,174 @@ void testResetJobCancelled() throws IOException {
       assertEquals(JobStatus.CANCELLED, updated.getStatus());
     }
 
+    private Job persistJobForTesting(String scope, JobConfig jobConfig, JobStatus status, LocalDateTime runDate) throws IOException, SQLException {
```

Comment:
This doesn't appear to be convenient in any way, as enqueueJob doesn't allow me to set the created/updated dates myself, which is the primary use case of this test scenario.  I've wrapped the persistJobForJobHistoryTesting() into the right section of the test case code, so it's clear what it goes with, but I definitely have to keep a separate method of creating job history in bulk.  If there's currently no use for that in the primary code, it may as well remain just in the limited test case scope, but it could get moved over if there's a real use case for it.